app/api/route.ts

```diff
@@ -1,6 +1,339 @@
+        sequence: 0,
+        extendedProperties: {
+          private: {
+            "X-MOZ-LASTACK": "20230815T225900Z",
+          },
+        },
+        eventType: "default",
+      },
+      {
+        kind: "calendar#event",
+        etag: '"3383036282414000"',
+        id: "nmmui7btr2h5uqpgpm030tjnkp_20230720T003000Z",
+        status: "confirmed",
+        htmlLink:
+          "https://www.google.com/calendar/event?eid=bm1tdWk3YnRyMmg1dXFwZ3BtMDMwdGpua3BfMjAyMzA3MjBUMDAzMDAwWiBjXzgxN3BzcDVlbWxnaDh2Z2c0MzJ1ZHJ0djIwQGc",
+        created: "2022-11-06T17:37:22.000Z",
+        updated: "2023-08-08T18:09:01.207Z",
+        summary: "SHO' & TELL",
+        description:
+          "It is time to Show and Tell!, hosted tonight by @thechamber69 grab the hdmi cable and show off what you've been up to! What you're working on isn't digital? WHO CARES, come by and show it off anyways!\u003cbr /\u003e\u003cbr /\u003e\u003cbr /\u003e\u003cbr /\u003eStarts at 7:30pm, hosted at Sea Cave, 3000 St. Claude.",
+        location: "Sea Cave, 3000 St Claude Ave, New Orleans, LA 70117, USA",
+        creator: {
+          email: "a@neworleansgamedev.com",
+        },
+        organizer: {
+          email: "c_817psp5emlgh8vgg432udrtv20@group.calendar.google.com",
+          displayName: "NOGD Events",
+          self: true,
+        },
+        start: {
+          dateTime: "2023-07-19T19:30:00-05:00",
+          timeZone: "America/Chicago",
+        },
+        end: {
+          dateTime: "2023-07-19T21:00:00-05:00",
+          timeZone: "America/Chicago",
+        },
+        recurringEventId: "nmmui7btr2h5uqpgpm030tjnkp",
+        originalStartTime: {
+          dateTime: "2023-07-19T19:30:00-05:00",
+          timeZone: "America/Chicago",
+        },
+        iCalUID: "nmmui7btr2h5uqpgpm030tjnkp@google.com",
+        sequence: 3,
+        extendedProperties: {
+          private: {
+            "X-MOZ-LASTACK": "20230816T232900Z",
+          },
+        },
+        eventType: "default",
+      },
+    ],
+  },
+];
+
+let cal: calendar_v3.Calendar;
+
+if (process.env.NODE_ENV == "production") {
+  cal = new calendar_v3.Calendar({ auth: process.env.CALENDAR_KEY });
+}
+
+// Get the latest week
+export async function GET(_: Request) {
+  if (process.env.NODE_ENV == "development") {
+    const dummyEvents: Event[] = dummyJSON.flatMap((item) =>
+      item.items.map((event: any) => {
+        return {
+          htmlLink: event.htmlLink,
+          summary: event.summary,
+          description: event.description,
+          location: event.location,
+          start: {
+            dateTime: event.start.dateTime,
+          },
+          end: {
+            dateTime: event.end.dateTime,
+          },
+        };
+      })
+    );
+
+    return NextResponse.json(dummyEvents, {
+      status: 200,
+    });
+  }
+
+  if (!process.env.CALENDAR_KEY) {
+    throw new Error("CALENDAR_KEY is not set");
+  }
+
+  const calendarIds = [
```

Comment:
this along with line 6 to 238 "dummyJson" could migrate to a file similar to the way we have "organizations.ts" if you don't mind.